src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -1005,6 +1005,7 @@ <h1>{{ _("Export transactions to CSV") }}</h1>
                     this.pageCount = 1;
                 } else {
                     this.tableHeader.classList.remove('hidden');
+                    this.feedbackRow.classList.remove('feedback-row');
```

Comment:
Not sure how this relates to the blue bar, also not necessary for the feedbackRow, we are adding `hidden` here already.

Reply:
Well, I noticed that the page originally starts from `class="hidden"` and then later `class='feedback-row'` is added and not removed again. It does cause the blue bar even though "hidden" is added.